src/App/index.js

```diff
@@ -16,23 +25,116 @@ class App extends Component {
     this.setState(({ mapVisible: prev }) => ({ mapVisible: !prev }));
   };
 
+  locationSuccess = pos => {
+    this.setState({
+      latitude: pos.coords.latitude,
+      longitude: pos.coords.longitude,
+      gotLocation: true,
+    });
+  };
+
+  locationError = err => {
+    console.warn(`ERROR(${err.code}): ${err.message}`);
+    this.setState({
+      locationError: true,
+    });
+    setTimeout(() => {
```

Comment:
What's the background on these setTimeouts? I'm always hesitant to commit these because of time dependency.

Reply:
Yes I agree, will remove.
background - 
One is so that the form can only be submitted once every 5 seconds. Others are error messages.

For the former I propose simply closing the form input as sort of debounce, requiring the user to toggle the form back on to submit another report. The error messages can be made dismissable.

Reply:
Okay cool. Totally down for things like temporary disabling, native to our code, but less so for external dependencies.